extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1538,6 +1545,35 @@ export class AmpA4A extends AMP.BaseElement {
    * @param {!Object<string, string|number>=} opt_extraVariables
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
+
+  /**
+   * @param {string=} headerValue Method as given in header.
+   */
+  getNonAmpCreativeRenderingMethod(headerValue) {
+    if (headerValue) {
+      return headerValue;
+    }
+    return Services.platformFor(this.win).isIos() ?
+        XORIGIN_MODE.SAFEFRAME : null;
+  }
+
+  /**
+   * Returns iframe attributes necessary to render creative in cross-domain
```

Comment:
This isn't the attributes on the iframe but rather the base object written to the name attribute

Reply:
Done.